tensorflow_addons/layers/esn.py

```diff
@@ -0,0 +1,209 @@
+            Default: True.
+        activation: Activation function to use.
+            Default: hyperbolic tangent (`tanh`).
+            If you pass `None`, no activation is applied
+            (ie. "linear" activation: `a(x) = x`).
+        kernel_initializer: Initializer for the `kernel` weights matrix,
+            used for the linear transformation of the inputs.
+            Default: `glorot_uniform`.
+        recurrent_initializer: Initializer for the `recurrent_kernel` weights matrix,
+            used for the linear transformation of the recurrent state.
+            Default: `glorot_uniform`.
+        bias_initializer: Initializer for the bias vector.
+            Default: `zeros`.
+        return_sequences: Boolean. Whether to return the last output.
+            in the output sequence, or the full sequence.
+        go_backwards: Boolean (default False).
+            If True, process the input sequence backwards and return the
+            reversed sequence.
+        unroll: Boolean (default False).
+            If True, the network will be unrolled,
+            else a symbolic loop will be used.
+            Unrolling can speed-up a RNN,
+            although it tends to be more memory-intensive.
+            Unrolling is only suitable for short sequences.
+
+    Call arguments:
+        inputs: A 3D tensor.
+        mask: Binary tensor of shape `(samples, timesteps)` indicating whether
+            a given timestep should be masked.
+        training: Python boolean indicating whether the layer should behave in
+            training mode or in inference mode. This argument is passed to the cell
+            when calling it. This is only relevant if `dropout` or
+            `recurrent_dropout` is used.
+        initial_state: List of initial state tensors to be passed to the first
+            call of the cell.
+     """
+
+    @typechecked
+    def __init__(
+        self,
+        units: TensorLike,
+        connectivity: FloatTensorLike = 0.1,
+        leaky: FloatTensorLike = 1,
+        spectral_radius: FloatTensorLike = 0.9,
+        use_norm2: bool = False,
+        use_bias: bool = True,
+        activation: Activation = "tanh",
+        kernel_initializer: Initializer = "glorot_uniform",
+        recurrent_initializer: Initializer = "glorot_uniform",
+        bias_initializer: Initializer = "zeros",
+        return_sequences=False,
+        go_backwards=False,
+        unroll=False,
+        **kwargs
+    ):
+        cell = ESNCell(
+            units,
+            connectivity=connectivity,
+            leaky=leaky,
+            spectral_radius=spectral_radius,
+            use_norm2=use_norm2,
+            use_bias=use_bias,
+            activation=activation,
+            kernel_initializer=kernel_initializer,
+            recurrent_initializer=recurrent_initializer,
+            bias_initializer=bias_initializer,
+            dtype=kwargs.get("dtype"),
+        )
+        super(ESN, self).__init__(
```

Comment:
We drop the support to python2 so using `super().__init__` is enough.

Reply:
Done!